packages/block-editor/src/components/media-placeholder/index.native.js

```diff
@@ -43,6 +43,9 @@ function MediaPlaceholder( props ) {
 		getStylesFromColorScheme,
 		multiple,
 		value = [],
+		children,
+		customEmptyStateContainerStyles,
```

Comment:
If we remove [the other prop](https://github.com/WordPress/gutenberg/pull/23833/files#r456253613) in favor of simplifying the props of this component, maybe we could rename `customEmptyStateContainerStyles` for something like `height` so we could just pass the `300` value for cover. Then we can merge this like we were doing: 

```
	[
		emptyStateContainerStyle,
		height && { height },
	],
```